packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -1,41 +1,127 @@
 @import '../../common/common';
 
-$rowHeight: 36px;
-$rowVerticalPadding: 10px;
+$rowHeight: 42px;
+$compactRowHeight: 32px;
```

Comment:
That's a reasonable feature request but is outside of the scope of this change. This is for adding a 'compact' variant to the design system, not opening the door to customizing to different sizes.